rxjs/src/app/app.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import {
+  combineLatest,
+  filter,
+  forkJoin,
+  map,
+  mergeMap,
+  Observable,
+  Subject,
+  Subscription,
+  switchMap,
+  debounceTime
```

Comment:
This import is a hint, so we need to stick into that. There were no mergeMap in the original task.